client/src/sampleData/activityData.js

```diff
@@ -1,134 +1,210 @@
+    feelings: [ 'surprised' ]
+  },
+  {
+    activityId: 29,
+    activity: 'Revel in the beauty of this incredible, ever-shifting world we live in.',
+    feelings: [ 'surprised' ]
+  },
+  {
+    activityId: 30,
+    activity: 'Find somewhere comfy and safe to hide.',
+    feelings: [ 'fearful' ]
+  },
+  {
+    activityId: 31,
+    activity: 'Grab a set of matches. Take a moment to hold what you are feeling in your mind, taking note of where it lives in your body.  Light the match. Feel the fear reach its limit as the fire reaches its peak, and blow out the fire and the fear is one sharp breath.',
+    feelings: [ 'fearful' ]
+  },
+  {
+    activityId: 32,
+    activity: 'Listen to some angry music and mosh around your living room.',
+    feelings: [ 'angry' ]
+  },
+  {
+    activityId: 33,
+    activity: "Write down the meanest, least generous thoughts you're having on a piece of paper. Let it all out. When you're done tear it into little pieces or crumple it up (whichever feels most satisfying) and toss the remnants in the bin outside your home.",
+    feelings: [ 'angry' ]
+  },
+  {
+    activityId: 34,
+    activity: 'Take a moment to give yourself some grace. Whatever you’re feeling is okay.',
+    feelings: [ 'disgusted' ]
+  },
+  {
+    activityId: 35,
+    activity: 'Sit under a tree and feel the trunk support you. Look up and watch the leaves sway in the breeze. What color is the sky today? How does the bark feel under your fingers?',
+    feelings: [ 'disgusted' ]
+  },
+  {
+    activityId: 36,
+    activity: 'Message someone you love to ask them to share something beautiful or good that happened in their day.',
+    feelings: [ 'sad' ]
+  },
+  {
+    activityId: 37,
+    activity: 'Get yourself some snacks, swaddle yourself in a comfy blanket, and either: listen to really sad music and feel your feelies, OR watch one of your favorite movies (sad or not, up to you).',
+    feelings: [ 'sad' ]
+  },
+  {
+    activityId: 38,
+    activity: 'Go outside and find a way to give a gift to a stranger. A compliment? A wink? A swipe of your transit card for a free ride? Anything goes.',
+    feelings: [ 'happy' ]
+  },
+  {
+    activityId: 39,
+    activity: 'Ride your bike to somewhere you love to get yourself a treat! On the way back try no handlebars!!',
+    feelings: [ 'happy' ]
+  },
+  {
+    activityId: 40,
+    activity: 'Set a 5, 10, or 15-minute timer and tidy your space in the easiest way possible. You can stop when it goes off even mid-task. Anything counts. Nice work!',
+    feelings: [ 'bad' ]
+  },
+  {
+    activityId: 41,
+    activity: 'Put on an irresistible song and wiggle your body the weirdest way you can. Decide on the movements based on listening to how your body wants to move, stretch, and shake out, NOT on if the dance moves are good. The more like Ace Ventura: Pet Detective meets hippie at Woodstock the better',
+    feelings: [ 'bad' ]
```

Comment:
Okay so y'alls activities are like so much better than mine. I might need to step up my self care game???